airgun/entities/user.py

```diff
@@ -49,7 +49,7 @@ def delete(self, entity_name):
         view = self.navigate_to(self, 'All')
         view.search(entity_name)
         view.table.row(username=entity_name)['Actions'].widget.click(handle_alert=True)
-        view.dialog.confirm()
+        self.browser.handle_alert()
```

Comment:
`handle_alert=True` being passed to the `click`  call above means this isn't necessary anymore.